pipes/WDL/tasks/tasks_interhost.wdl

```diff
@@ -492,42 +492,112 @@ task merge_vcfs_gatk {
     }
   }
 
-  command {
+  command <<<
 
     # tabix index input vcfs (must be gzipped)
     parallel -I ,, \
       "tabix -p vcf ,," \
-      ::: "${sep=' ' in_vcfs_gz}"
+      ::: "~{sep=' ' in_vcfs_gz}"
 
     # index reference to create .fai and .dict indices
-    samtools faidx "${ref_fasta}"
-    picard CreateSequenceDictionary R="${ref_fasta}" O=$(basename $(basename "${ref_fasta}" .fasta) .fa).dict
+    samtools faidx "~{ref_fasta}"
+    picard CreateSequenceDictionary R="~{ref_fasta}" O=$(basename $(basename "~{ref_fasta}" .fasta) .fa).dict
 
     # store input vcf file paths in file
-    for invcf in $(echo "${sep=' ' in_vcfs_gz}"); do 
+    for invcf in $(echo "~{sep=' ' in_vcfs_gz}"); do 
       echo "$invcf" > input_vcfs.list
     done
 
     # merge
-    gatk3 -T CombineVariants -R "${ref_fasta}" -V input_vcfs.list -o "${output_prefix}.vcf" -genotypeMergeOptions UNIQUIFY
+    gatk3 -T CombineVariants -R "~{ref_fasta}" -V input_vcfs.list -o "~{output_prefix}.vcf" -genotypeMergeOptions UNIQUIFY
     
     # bgzip output
-    bgzip "${output_prefix}.vcf"
+    bgzip "~{output_prefix}.vcf"
 
     # tabix index the vcf to create .tbi file
-    tabix -p vcf "${output_prefix}.vcf.gz"
-  }
+    tabix -p vcf "~{output_prefix}.vcf.gz"
+  >>>
 
   output {
-    File merged_vcf_gz     = "${output_prefix}.vcf.gz"
-    File merged_vcf_gz_tbi = "${output_prefix}.vcf.gz.tbi"
+    File merged_vcf_gz     = "~{output_prefix}.vcf.gz"
+    File merged_vcf_gz_tbi = "~{output_prefix}.vcf.gz.tbi"
   }
 
   runtime {
-    docker: "${docker}"
+    docker: docker
     memory: select_first([machine_mem_gb, 3]) + " GB"
     cpu: 2
     dx_instance_type: "mem1_ssd1_v2_x2"
     maxRetries: 2
   }
 }
+
+task reconstructr {
+  input {
+    File         msa_fasta
+    File         ref_fasta
+    File         date_csv
+    File         depth_csv
+    Array[File]+ lofreq_vcfs
+    Int          n_iters
+
+    String       out_basename = "reconstructR"
+
+    Int          cpus = 8
+    Int          machine_mem_gb = 15
+    Int          disk_size = 375
+    String       docker = "ghcr.io/broadinstitute/reconstructr:main"
+  }
+
+  command <<<
+    set -e -o pipefail
+
+    # stage input files
+    mkdir -p input_data input_data/vcf input_data/coverage
+    /opt/reconstructR/scripts/cp_and_decompress.sh "~{msa_fasta}" input_data/aligned.fasta
```

Comment:
Can these be put on the PATH in the reconstructr container?

Reply:
done!